### PR TITLE
fix: fix pci mobile sidebar

### PR DESCRIPTION
**Pull request by marie-j**
MANAGER-2971

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/sidebar/sidebar.service.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ export default class Sidebar {
     return [
       ...MENU.map(item => ({
         name: item.translation,
-        title: this.$translate.instant(item.translation),
+        title: item.title,
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/sidebar/sidebar.service.js]: What does this have to do with the schmilblick ?

Comment (marie-j) [reply packages/manager/modules/pci/src/projects/project/sidebar/sidebar.service.js]: Long story short, sidebar not handled the same way on mobile, I've started a refactor following that issue  but in the meanwhile
